### ISSUE TITLE
WARNING: 'bundle' has been deprecated. Use 'build' instead.

### ISSUE THREAD

**Issue by abachuk** [arunoda/meteor-up-legacy#136]
I was able to do deploys before, now with meteor 0.9.2 and the latest version of mup, ran mup update and setup, and getting this error: 

```
bundling Error: code=8, error:
-------------------STDOUT-------------------
WARNING: 'bundle' has been deprecated. Use 'build' instead.
```

**Comment by arunoda** [comment]
Interesting. I'm out today. Will work on once I came back.

On Saturday, September 20, 2014, Alex Bachuk notifications@github.com
wrote:

> I was able to do deploys before, now with meteor 0.9.2 and the latest
> version of mup, ran mup update and setup, and getting this error:
> 
> bundling Error: code=8, error:
> -------------------STDOUT-------------------
> WARNING: 'bundle' has been deprecated. Use 'build' instead.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/arunoda/meteor-up/issues/136.

## 

---

Arunoda Susiripala
I curate Meteor Weekly - Check it out!
http://meteorhacks.com/meteor-weekly/?utm_source=email-footer&utm_medium=email&utm_campaign=meteorweekly

**Comment by arunoda** [comment]
Check now. Just released a new version: 0.6.7

---

Arunoda Susiripala
I curate Meteor Weekly - Check it out!
http://meteorhacks.com/meteor-weekly/?utm_source=email-footer&utm_medium=email&utm_campaign=meteorweekly

**Comment by abachuk** [comment]
Thank you. It works now.
